source/screens/Home/ActionsPane/ActionsPane.tsx

```diff
@@ -1,19 +1,40 @@
-import React from 'react';
+import React, {useState} from 'react';
+import {useNavigate} from 'react-router-dom';
 
 import './ActionsPane.scss';
+import {Modal} from 'semantic-ui-react';
+import ReceiveModal from '../../../components/ReceiveModal';
+
+const ActionsPane: React.FC = () => {
+  const [showReceive, setShowReceive] = useState(false);
+  const navigate = useNavigate();
+  const handleNavigateToSend = (): void => {
+    navigate('/send');
+  };
 
-const ActionsPane = () => {
   return (
     <div className="actions-pane">
-      <button
-        type="button"
-        className="actions-pane__button actions-pane__button--receive"
+      <Modal
+        onClose={(): void => setShowReceive(false)}
+        onOpen={(): void => setShowReceive(true)}
+        open={showReceive}
+        className="modal"
+        trigger={
+          <button
+            type="button"
+            className="actions-pane__button actions-pane__button--receive"
+          >
+            Receive
+          </button>
+        }
       >
-        Receive
-      </button>
+        <ReceiveModal onClose={(): void => setShowReceive(false)} />
+      </Modal>
+
```

Comment:
Please disable "Send" button by default and enable only when `const { initialized } = useWalletSyncProgress()` becomes true.